Bleedify/BleedifyServices/PersonalService.cs

```diff
@@ -0,0 +1,26 @@
+﻿using BleedifyModels.ModelsEF;
+using BleedifyModels.Repositories;
+using BleedifyModels.Validators;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BleedifyServices
+{
+    public class PersonalService
+    {
+        private IRepository<int, Personal> _repository;
+
+        public PersonalService()
+        {
+            _repository = new PersonalRepository(new PersonalValidator());
+        }
+        
+        public void Register(Personal personal)
+        {
+            _repository.Add(personal);
```

Comment:
Register practic e un add, pana aici e ok.
Doar sa mai adauci si o verificare suplimentara -> sa nu aibe deja cont, daca are deja cont arunca exceptie ceva gen ->
"Utilizizatorul X deja are cont" sau ceva.
Dfiind ca esti primul, te rog sa faci si o clasa ServiceException,  exceptia de care iti ziceam anterior fiind ServiceException.

P.S. E cam acelasi cod ca la ValidationExeption dar ne ajuta sa ne dam seama unde e problema in cazul in care ceva nu e ok.